docs/contract-updatability.md

```diff
@@ -0,0 +1,357 @@
+---
+title: Contract Updatability
+---
+
+## Introduction
+A [contract](https://docs.onflow.org/cadence/language/contracts/) in Cadence is a collection of data (its state) and
+code (its functions) that lives in the contract storage area of an account.
+When a contract is updated, it is important to make sure that the changes introduced does not lead to runtime
+inconsistencies for already persisted data.
```

Comment:
We normally use the terms "storage" and "stored", instead of "persistence" and "persisted"

```suggestion
inconsistencies for already stored data.
```